packages/store/src/workspace/page.ts

```diff
@@ -276,6 +276,28 @@ export class Page extends Space<PageData> {
     this.updateBlock(model, props);
   }
 
+  moveBlock(
+    model: BaseBlockModel,
+    oldParent: BaseBlockModel,
+    oldIndex: number,
+    newParent: BaseBlockModel,
+    newIndex: number
+  ) {
```

Comment:
Lots of positional arguments of the same types is usually a code smell. I'll defer to @doodlewind and other code owners for whether this is actually aligned with the codebase's maintainability guidelines.

Alternatively, pass in an
```
  opts: {
    target: BaseBlockModel,
    from: { parent: BaseBlockModel, idx: number },
    to: { parent: BaseBlockModel, idx: number }
  }
```